thoth_user_api/swagger.yaml

```diff
@@ -606,3 +612,43 @@ definitions:
           type: string
           description: Value of the supplied environment variable.
           example: '1'
+  Metadata:
```

Comment:
why is it uppercase?

Reply:
Definitions start with Capital Letter for other definitions, so followed the same standard.